### PR TITLE
ci: update job names

### PR DESCRIPTION
**Pull request by umarcor**
This PR is for changing CI job names. No functional modification.

### CODE REVIEW

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -64,10 +64,12 @@ jobs:
   Linux:
     strategy:
       matrix:
-        python-version: [ 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38', 'cp39-cp39' ]
+        python-version: [ '6', '7', '8', '9' ]
       fail-fast: false
 
-    name: ${{ matrix.python-version }} • manylinux
+    env:
+      PYVER: ${{ format('cp3{0}-cp3{0}m', matrix.python-version) }}
```
Comment (mithro) [inline .github/workflows/publish-to-pypi.yml:71]: `cp38-cp38` and `cp39-cp39` don't end in an `m`.

File: .github/workflows/publish-to-pypi.yml [hunk 2]
```diff
@@ -64,10 +64,16 @@ jobs:
   Linux:
     strategy:
       matrix:
-        python-version: [ 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38', 'cp39-cp39' ]
+        py-ver:
```
Comment (mithro) [inline .github/workflows/publish-to-pypi.yml:67]: Maybe just be more explicit here like this?
```suggestion
        python-version:
          - { name: '3.6', manylinux: 'cp36-cp36m' }
          - { name: '3.9', manylinux: 'cp39-cp39m' }
```

Comment (umarcor) [reply .github/workflows/publish-to-pypi.yml]: Indeed.... Done!

**Review by mithro — APPROVED** [review]
Thank you so much!

**Pull request merged**